### PR TITLE
Add global sort related docs.

### PR DESCRIPTION
**Pull request by ran-huang**
### First-time contributors' checklist 

- [X] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [X] master (the latest development version)
- [X] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/14801
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-global-sort.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+> **Warning:**
+>
+> 该功能目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
+
+## 功能概览
+
+TiDB 全局排序功能增强了数据导入和 DDL（数据定义语言）操作的稳定性和执行效率。全局排序作为[分布式执行框架](/tidb-distributed-execution-framework.md)中的通用算子，通过分布式执行框架，在云上提供全局排序服务。
+
+你可以轻松扩展全局排序来支持多种共享存储接口，例如 S3 和 POSIX，实现与不同存储系统的无缝集成。全局排序功能可以灵活地为各种使用场景提供高效和适配的数据排序服务。
```
Comment (Benjamin2037) [inline tidb-global-sort.md:16]:
```suggestion
后续我们会扩展全局排序来支持多种共享存储接口，例如 S3 和 POSIX，实现与不同存储系统的无缝集成。全局排序功能可以灵活地为各种使用场景提供高效和适配的数据排序服务。
```

File: tidb-global-sort.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+## 目标
+
+全局排序功能旨在提高 `IMPORT INTO` 和 `CREATE INDEX` 的效率。通过将全局排序集成到后端任务中，可以提高稳定性、可控性和可扩展性，提供更好的用户体验和更高质量的服务。
+
+全局排序功能在统一的分布式并行执行框架中执行任务，保证数据在全局范围内的高效并行排序。
```
Comment (Benjamin2037) [inline tidb-global-sort.md:22]:
```suggestion
全局排序功能在后端任务分布式并行执行框架中执行任务，保证数据在全局范围内的高效并行排序。
```

File: tidb-global-sort.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+## 限制
+
+目前，全局排序功能无法用于查询过程中对查询结果进行排序。
```
Comment (Benjamin2037) [inline tidb-global-sort.md:26]:
```suggestion
目前，全局排序功能不支持用于查询过程中对查询结果进行排序。
```

File: tidb-global-sort.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+## 功能概览
+
+TiDB 全局排序功能增强了数据导入和 DDL（数据定义语言）操作的稳定性和执行效率。全局排序作为[分布式执行框架](/tidb-distributed-execution-framework.md)中的通用算子，通过分布式执行框架，在云上提供全局排序服务。
+
+你可以轻松扩展全局排序来支持多种共享存储接口，例如 S3 和 POSIX，实现与不同存储系统的无缝集成。全局排序功能可以灵活地为各种使用场景提供高效和适配的数据排序服务。
+
+## 目标
+
+全局排序功能旨在提高 `IMPORT INTO` 和 `CREATE INDEX` 的效率。通过将全局排序集成到后端任务中，可以提高稳定性、可控性和可扩展性，提供更好的用户体验和更高质量的服务。
```
Comment (Benjamin2037) [inline tidb-global-sort.md:20]:
```suggestion
全局排序功能旨在提高 `IMPORT INTO` 和 `CREATE INDEX` 的稳定性与效率。通过将任务需要处理的数据进行全局排序，可以提高数据写入 TiKV 的稳定性、可控性和可扩展性，提供更好的数据导入与 DDL 任务的用户体验及更高质量的服务。
```

File: tidb-global-sort.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+全局排序功能的算法如下图所示：
+
+![全局排序功能算法](/media/dist-task/global-sort.jpeg)
+
+详细的实现原理如下：
+
+### 第 1 步：扫描和准备数据
+
+1. TiDB 节点扫描特定范围的数据后：
```
Comment (Benjamin2037) [inline tidb-global-sort.md:54]:
```suggestion
1. TiDB 节点扫描特定范围的数据后（数据源可以是 CSV 数据或者 TiKV 中的表数据）：
```

File: tidb-global-sort.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+    1. TiDB 节点将扫描的数据编码为 Key-Value 对。
+    2. TiDB 节点将 Key-Value 对排序为多个块数据段，每个段是一个文件，并将这些文件上传到云存储中。
```
Comment (Benjamin2037) [inline tidb-global-sort.md:57]:
```suggestion
    2. TiDB 节点将 Key-Value 对排序为多个块数据段（数据段局部有序），每个段是一个文件，并将这些文件上传到云存储中。
```

File: tidb-global-sort.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+
+2. TiDB 节点记录了每个段的连续实际键值范围（被称为统计信息文件），这是可扩展排序实现的关键准备工作。这些文件随实际数据一起上传到云存储中。
+
+### 第 1 步：排序和分发数据
```
Comment (Benjamin2037) [inline tidb-global-sort.md:61]:
```suggestion
### 第 2 步：排序和分发数据
```

File: system-variables.md [hunk 8]
```diff
@@ -1261,6 +1261,20 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
...
 - 该变量由 `tidb_ddl_distribute_reorg` 改名而来。
 
+### `tidb_cloud_storage_uri` <span class="version-mark">从 v7.4.0 版本开始引入</span>
+
+> **警告：**
+>
+> 该变量目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
+
+- 作用域：GLOBAL
+- 是否持久化到集群：是
```
Comment (Oreoxmt) [inline system-variables.md:1271]:
```suggestion
- 是否持久化到集群：是
- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
```

File: tidb-global-sort.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+
+> **Warning:**
+>
+> 该功能目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
+
+## 功能概览
+
+TiDB 全局排序功能增强了数据导入和 DDL（数据定义语言）操作的稳定性和执行效率。全局排序作为[分布式执行框架](/tidb-distributed-execution-framework.md)中的通用算子，通过分布式执行框架，在云上提供全局排序服务。
+
+后续我们会扩展全局排序来支持多种共享存储接口，例如 S3 和 POSIX，实现与不同存储系统的无缝集成。全局排序功能可以灵活地为各种使用场景提供高效和适配的数据排序服务。
```
Comment (Oreoxmt) [inline tidb-global-sort.md:16]:
```suggestion
TiDB 计划未来扩展全局排序来支持多种共享存储接口，例如 Amazon S3 和 POSIX，实现与不同存储系统的无缝集成。全局排序功能可以灵活地为各种使用场景提供高效且适配的数据排序服务。
```

File: tidb-global-sort.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+
+## 目标
+
+全局排序功能旨在提高 `IMPORT INTO` 和 `CREATE INDEX` 的稳定性与效率。通过将任务需要处理的数据进行全局排序，可以提高数据写入 TiKV 的稳定性、可控性和可扩展性，提供更好的数据导入与 DDL 任务的用户体验及更高质量的服务。
+
+全局排序功能在后端任务分布式并行执行框架中执行任务，保证后端任务所需处理的数据在全局范围内的有序。
```
Comment (Oreoxmt) [inline tidb-global-sort.md:22]:
```suggestion
全局排序功能旨在提高 `IMPORT INTO` 和 `CREATE INDEX` 的稳定性与效率。通过将任务需要处理的数据进行全局排序，可以提高数据写入 TiKV 的稳定性、可控性和可扩展性，从而提供更好的数据导入与 DDL 任务的用户体验及更高质量的服务。

全局排序功能在后端任务分布式并行执行框架中执行任务，确保后端任务所需处理的数据在全局范围内保持有序。
```

File: tidb-global-sort.md [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+
+## 限制
+
+目前，全局排序功能不支持用于查询过程中对查询结果进行排序。
```
Comment (Oreoxmt) [inline tidb-global-sort.md:26]:
```suggestion
目前，全局排序功能不支持在查询过程中对查询结果进行排序。
```

File: tidb-global-sort.md [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
+
+## 使用方法
+
+要开启全局排序功能，执行以下步骤：
+
+1. 将 [`tidb_enable_dist_task`](/system-variables.md#tidb_enable_dist_task-从-v710-版本开始引入) 的值设置为 `ON`，开启分布式执行框架：
```
Comment (Oreoxmt) [inline tidb-global-sort.md:32]:
```suggestion
1. 将 [`tidb_enable_dist_task`](/system-variables.md#tidb_enable_dist_task-从-v710-版本开始引入) 的值设置为 `ON`，以开启分布式执行框架：
```

File: tidb-global-sort.md [hunk 13]
```diff
@@ -0,0 +1,67 @@
...
+详细的实现原理如下：
+
+### 第 1 步：扫描和准备数据
+
+1. TiDB 节点扫描特定范围的数据后（数据源可以是 CSV 数据或者 TiKV 中的表数据）：
+
+    1. TiDB 节点将扫描的数据编码为 Key-Value 对。
+    2. TiDB 节点将 Key-Value 对排序为多个块数据段（数据段局部有序），每个段是一个文件，并将这些文件上传到云存储中。
+
+2. TiDB 节点记录了每个段的连续实际键值范围（被称为统计信息文件），这是可扩展排序实现的关键准备工作。这些文件随实际数据一起上传到云存储中。
```
Comment (Oreoxmt) [inline tidb-global-sort.md:59]:
```suggestion
    1. TiDB 节点将扫描的数据编码为键值对。
    2. TiDB 节点将键值对排序为多个块数据段（数据段局部有序），每个段是一个文件，并将这些文件上传到云存储中。

2. TiDB 节点记录了每个段的连续实际键值范围（称为统计信息文件），这是可扩展排序实现的关键准备工作。这些文件随实际数据一起上传到云存储中。
```

File: tidb-global-sort.md [hunk 14]
```diff
@@ -0,0 +1,67 @@
...
+
+### 第 2 步：排序和分发数据
+
+从第一步中，全局排序的程序获取了一个已排序块的列表及其对应的统计信息文件，这些文件记录了本地已排序块的数量。此外，全局排序程序还记录了一个实际数据范围，可以由 PD 用于拆分和打散。接下来将执行以下步骤：
+
+1. 将统计信息文件中的记录排序，将其分成大小相近的范围，这些范围是将并行执行的子任务。
```
Comment (Oreoxmt) [inline tidb-global-sort.md:65]:
```suggestion
从第一步中，全局排序的程序获取了一个已排序块的列表及其对应的统计信息文件，这些文件记录了本地已排序块的数量。此外，全局排序程序还记录了一个实际数据范围，供 PD 用于数据拆分和打散。接下来将执行以下步骤：

1. 将统计信息文件中的记录排序，划分为大小相近的范围，每个范围将作为一个并行执行的子任务。
```

File: tidb-global-sort.md (hunk 9 above)
Comment (Oreoxmt) [inline tidb-global-sort.md:16]: 在 7.4 中全局排序支持 S3 吗？下面示例使用的是 S3

https://github.com/pingcap/docs-cn/blob/ffedfcdf0cabac9dd5df2fd8ffc9cc9b12fa7a18/tidb-global-sort.md?plain=1#L38-L42

Comment (Benjamin2037) [reply tidb-global-sort.md:16]: 7.4 目前只正式支持了 AWS 的 S3 存储。

Comment (ran-huang) [reply tidb-global-sort.md:16]: 建议改成这样
```suggestion
全局排序目前仅支持使用 Amazon S3 作为云存储，未来将扩展支持多种共享存储接口，例如 POSIX，实现与不同存储系统的无缝集成。全局排序功能可以灵活地为各种使用场景提供高效且适配的数据排序服务。
```

**Review by Oreoxmt — APPROVED** [review]

**Review by Benjamin2037 — APPROVED** [review]
LGTM

**Pull request merged**